static/src/javascripts/projects/common/modules/onward/tech-feedback.js

```diff
@@ -116,6 +117,10 @@ define([
      * override those at the time the email is sent.
      */
     return function () {
+        detect.adblockInUse.then(function(adblockInUse){
+            adblockBeingUsed = adblockInUse;
```

Comment:
You probably need to make the whole chain use promises now this is async, otherwise your `adblockBeingUsed` var is always going to be `false` - the functionality below needs to be inside this `then`, right?